package/src/wallets/keplr/KeplrWallet.ts

```diff
@@ -165,29 +271,36 @@ export class KeplrWallet implements Wallet {
       msgs,
       {
         amount: [],
-        gas: '150000',
+        // this is not good
+        gas: '250000',
```

Comment:
todo @alecande11

Reply:
It's a bit complicated to calculate the gas needed for each tx, and it also slow down the transfer, so since they are all the same type I just use a constant value

Reply:
feels dangerous, as gas is sometimes dependent on network load. So simulation to check the msg is even correct makes sense anyway.

But you can decide, the ICS20 transfer takes a bit more gas and I can leave it hardcoded as some kind of switch based on operation type.